### ISSUE TITLE
Hearts all over 

### ISSUE THREAD

**Issue by Shinidk** [Sqeegie/MoreHearts#1]
Hey,
I use this plugin for my RPG server but players have been complaining that the hearts fill up most of their screen so i would like to know if you could add an option or if there is, that you can add all the hearts you want but will only show default. I tried to set it to false but did not work

**Comment by Sqeegie** [comment]
There's currently a option known as "HideHearts" that theoretically should only show the default heart amount. However, as far as I know, it's never worked, even in the original plugin. I've known about this bug, and have already added it to be patched in the next update. Now that I know someone actually needs it fixed, I'll prioritize finishing this next update, and get it out to you as soon as possible.

**Comment by Shinidk** [comment]
Sqeegie I appreciate it and I would like to ask if I paid you something like 5-10 USD would you be able to finish it soon and add 1 feature ? The feature would be: Players Will be able to see the amount of life they have in their MC exp bar or something like that and if possible would you be able to create something like a Health Bar só people could see the health you have? I'm whiling to pay for it if you think creating it will be wasting your time.

**Comment by Sqeegie** [comment]
I was planning on trying to finish the patch today after I get back from collage. As for the additional feature, I'm willing to implement it. I'll need to think it over more closely to see what it entails. If I don't find any major issues with it, I'll see what I can do with implementing it. No need for payment (Unless it becomes a major pain in butt. 😁)

**Comment by Shinidk** [comment]
Here is a link to the Health bar that I mean: 
https://www.spigotmc.org/resources/❤-simple-health-bar-❤-supports-1-9.18413/
It would be awesome if you could do it! My server opens in 5 days so even better if before that hahahaha. 
Also do you make custom plugins ? If yes maybe you could make 1 for me for payment ? If yes send me your skype and I will add you so we can discuss it better.

**Comment by Sqeegie** [comment]
Alright, I published the patch. I also added a basic health bar for other players. I don't have any indicator for yourself yet, except for the default /hearts, or /h command. I'll probably expand onto this feature sometime in the future when I have time. However, I'm leaving on a trip this weekend, and I wanted to get the basic version out before I leave.

I've only done some basic testing, so if you run into another bug, be sure to let me know.

As for custom plugins, I might be interested in making one. It depends on what it is, and how complex/large it is. I don't exactly want to post my Skype name here, so I'll have to get it to you another way.
